README.md

```diff
@@ -1 +1,180 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Weboack <a name="about-project"></a>
+
+**Leaderboard** list app is a project to act as a score board for a computer game.
+
+## 🛠 Built With <a name="built-with"></a>
+- Webpack
+- Html
+- Css
+- Javascript
+- Gitflow
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Markup</summary>
+  <ul>
+    <li><a href="https://html5.org/">HTML5</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Interaction</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Easy navigation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need an IDE such as VS code, sublime, atom e.t.c:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+ $ cd my-folder
+ $ git@github.com:Cisco-the-wayword/Leaderboard.git
+```
+
+### Install
+
+Install git to work on this project with:
+
+```sh
+   $ git config --global user.name "First Last"
+
+   $ git config --global user.email "example@example.com"
+```
+
+### Usage
+
+To run the project have Node.js installed, execute the following command in your terminal:
+
+```sh
+  $ npm start
+```
+
+
+### Run tests
+
+To run tests, run the HTML document in Google Chrome and right-click the HTML element you want to inspect. Click on "Inspect" and you will have the tools to run, analyze, and even debug the code
```

Comment:
- Hey, since you won't be adding testing for this project, please, kindly remove this section from the readme body and from the top navigation to improve your readme.